packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -293,11 +330,15 @@ export const moduleSelected: Action<{
       );
     }
 
+    if (module.shortid === state.editor.currentModuleShortid) {
```

Comment:
module can probably be null here?

Reply:
Hm, if it is `null` here I would state it is a bug. Cause with `path` (from VSCode) it will throw an error above and it is caught. With an `id` it is actually you selecting the file in the explorer, so it has to be there